src/aks-preview/HISTORY.rst

```diff
@@ -13,7 +13,11 @@ Pending
 +++++++
 * Update to use 2022-04-02-preview api version.
 
-0.5.67 (NOT RELEASED)
+0.5.68
```

Comment:
Li ma added a feature about api and released 0.5.57, need to resolve the conflicts here.

Reply:
Yes. Updated.